pytest_fixtures/component/leapp_client.py

```diff
@@ -167,6 +167,7 @@ def custom_leapp_host(upgrade_path, module_target_sat, module_sca_manifest_org,
         host_class=ContentHost,
         deploy_rhel_version=upgrade_path['source_version'],
         deploy_flavor=settings.flavors.default,
+        deploy_network_type='ipv6' if settings.server.is_ipv6 else 'ipv4',
```

Comment:
Great, PRT passed!
Could you also add it here?
https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/ui/test_errata.py#L876

Reply:
Nice! I wonder why it's failing locally.

Reply:
Added.